theme/static/style/style.css

```diff
@@ -432,6 +476,39 @@ pre code {
         padding-left: 1.8rem;
         text-align: left;
     }
+
+    /*features page*/
+    .features .w50 {
+        padding-top: 2rem;
+    }
+    .features .col-8 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features .col-3 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features img {
+        min-width: 95%;
+        max-width: 95%;
+        float: none;
+        margin: auto;
+        display: block;
+    }
+    .features li {
+        padding-left: 1.8rem;
+        padding-right: 1.8rem;
+    }
+    .w25 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .testplan {
```

Comment:
Prefix with `.features` just for consistency with the rest and for better clarity.